src/web/components/Lazy.tsx

```diff
@@ -1,22 +1,15 @@
 import React from 'react';
 import { css } from 'emotion';
 
+import isChromatic from 'storybook-chromatic/isChromatic';
```

Comment:
https://bundlephobia.com/result?p=storybook-chromatic@3.5.2 ok

Reply:
I might change this approach based on [this gist](https://gist.github.com/kylesuss/a757e12e3f3dd9c9be3ed083c6c1313c) from the Chromatic people which looks neater